projects/flatbuffers/project.yaml

```diff
@@ -4,6 +4,8 @@ primary_contact: "derekbailey@google.com"
 auto_ccs:
   - "Adam@adalogics.com"
```

Comment:
can you nix these two, i don't know the first and haven't heard from `vglavnyy` in ages